client/src/utils/consts.js

```diff
@@ -9,3 +11,11 @@ export const MUI_INPUT_TYPES = [
 ];
 
 export const MUI_PLACEMENTS = ['left', 'right', 'top', 'bottom'];
+
+export const DB_AUTH_URL = 'http://localhost:3000/auth';
+
+export const DB_DOGS_URL = 'http://localhost:3000/dogs';
+
+export const DogsProxy = new ProxyDB(DB_DOGS_URL);
```

Comment:
for debug add this to window object , then go to console and test your proxy 
like: 
`window.DogsProxy  = DogsProxy;`

in console run: 
`DogsProxy.getData().then(console.log).catch(console.error)`